compiler/modules/bir/bir.bal

```diff
@@ -124,6 +124,16 @@ public function createBasicBlock(FunctionCode code, string? name = ()) returns B
     return bb;
 }
 
+public function discardBasicBlocksUpTo(FunctionCode code, BasicBlock upTo) {
```

Comment:
Need comment or name change to make it clear whether up to is inclusive or exclusive.

I think you can do this more simply as something like `code.blocks.setLength(upTo.label)`.

Reply:
Also I think UpTo is confising, because you are discarding from the end backwards to.  So I think it's `discardBasicBlocksFrom` i.e. from this block onwards.

Reply:
This makes the use in expr.bal easier to understand.